notebooks/Project and Developer Similarity.ipynb

```diff
@@ -0,0 +1,625 @@
+{
```

Comment:
<p>I really like the cooking analogy and the language! </p><p> </p><p>May be we could take it one step further by garnishing the story with emojis related to the topic 👨 🍳 = 👨‍🍳 </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mloncode/devfest2019-workshop/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Completely up for that</p>